### ISSUE TITLE
Single line function fix

### ISSUE THREAD

**Issue by kardasis** [testdouble/standard#309]
I have a file called `singlelinefunction.rb` with the following contents
```
def a() return ('hi' == 'hello') end
def b() end
```
When I run `bundle exec standardrb singlelinefunction.rb --fix`, the resulting file is
```
def a (
  
  "hi" == "hello")
end

def b
end
```
Which is obviously incorrect.

When I remove the second function definition, so the contents of the file is 
```
def a() return ('hi' == 'hello') end
```
Then the output is 
```
def a
  ("hi" == "hello")
end
```
Which seems like bad style but at least is correct ruby code.

**Comment by kardasis** [comment]
Also worth noting that if the second function has a non-empty body, i.e. `def b 'yo' end`, then the result is the same as when there's no function there at all.  So it appears that the empty function body is at least partly responsible here.

**Comment by searls** [comment]
@kardasis Looks like @koic fixed this in the latest Rubocop release, so a fix should land in the next version we cut of Standard

**Comment by koic** [comment]
@searls I'm sorry I haven't been able to share. This issue has not been resolved yet. I will let you know when it is resolved.

**Comment by kardasis** [comment]
@koic So this was not addressed in rubocop yet?  Should I make an issue over there?

**Comment by koic** [comment]
@kardasis I opened https://github.com/rubocop/rubocop/pull/9926 to resolve this issue. Thank you.

**Comment by koic** [comment]
@searls This issue has been resolved with the release of RuboCop 1.18.4.
https://github.com/rubocop/rubocop/releases/tag/v1.18.4

**Comment by searls** [comment]
Thanks @koic, @camilopayan released this update last night!
